service/post.service.js

```diff
@@ -74,6 +75,34 @@ exports.createComment = async (author, text, postId) => {
   return post;
 };
 
+exports.reactLove = async (author, postId) => {
+  const post = await Post.findOneAndUpdate(
+    {
+      _id: postId,
+    },
+    {
+      $push: {
+        loves: {
+          author: author
+        },
+      },
+    },
+    {
+      new: true,
+    }
+  );
+
+  const data = {
+    username: author,
+    summary: `reacted love on ${post.author}'s post`,
+    extraText: text.substring(0, 50),
```

Comment:
As there is no `body` in the request we cannot get the `text`, please remove it.